### DISCUSSION TITLE
media-settings-* not available in React package

### DISCUSSION THREAD

**Discussion by wesbos** [muxinc/media-chrome#1037] [Q&A] [answered]
I don't know if this matters as web components just work in React now, but the `<media-settings-menu>` component and all the related menu items are not exported or available in the React package.

**Comment by luwes** [comment, marked as answer]
we decided to put them in a separate import since version 3 or 4 because they add some 10-20kB gzip or so.

```js
import {
  MediaSettingsMenu,
  MediaSettingsMenuButton,
  MediaSettingsMenuItem,
  MediaCaptionsMenu,
  MediaPlaybackRateMenu,
} from "media-chrome/dist/react/menu";
```

(seems something is up with the import path, without the `dist` it was complaining)

I quickly put together a Codesandbox https://codesandbox.io/p/sandbox/media-chrome-settings-menu-react-ks8cyw?file=%2Fsrc%2FApp.tsx

hope this helps and thanks for the shoutout on Syntax!! ❤

**Comment by wesbos** [comment]
awesome thanks a ton! Might be worth adding to the docs?

**Comment by wesbos** [comment]
In case anyone is reading this - the exports are available at `import { MediaSettingsMenu, MediaSettingsMenuButton, MediaSettingsMenuItem, MediaCaptionsMenu, MediaPlaybackRateMenu, MediaRenditionMenu } from 'media-chrome/react/menu';`
